app/journal/journal-client.tsx

```
@@ -0,0 +1,252 @@
+                return aVal > bVal ? 1 : -1;
+            } else {
+                return aVal < bVal ? 1 : -1;
+            }
+        });
+
+        setFilteredTrades(filtered);
+    }, [trades, searchQuery, filterStrategy, sortBy, sortOrder, showHidden]);
+
+    const validateTradeForm = (trade: typeof newTrade) => {
+        if (!trade.symbol) return "Symbol is required";
+        if (!trade.entryPrice || isNaN(parseFloat(trade.entryPrice))) return "Valid entry price is required";
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)